modules/UI/invite/Invite.js

```diff
@@ -48,11 +50,6 @@ class Invite {
         APP.UI.addListener( UIEvents.INVITE_CLICKED,
                             () => { this.openLinkDialog(); });
 
-        APP.UI.addListener( UIEvents.INVITE_URL_INITIALISED,
```

Comment:
Fixed in https://github.com/jitsi/jitsi-meet/commit/d342f93547ee52e603f78c3e80c7765fb0f908bc